modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb

```diff
@@ -63,8 +63,8 @@ def perform(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
 
         raise e
       rescue ::Common::Exceptions::BackendServiceException => e
-        set_errored_state_on_claim(auto_claim)
-        error_message = get_error_message(e)
+        error_message = auto_claim.evss_response&.dig(0, 'text') if auto_claim.evss_response.present?
```

Comment:
Same as above, plus, I believe on the first run through this would also be empty because the `evss_response` is not set yet, so then the log would not have any data for that (maybe ?), so the order would need to change if you wanted to keep it in here (swap line 66 and 67 I _believe_).